app/typings/mri.d.ts

```diff
@@ -0,0 +1,32 @@
+declare module 'mri' {
```

Comment:
Is there a particular reason why this is added here rather than to `app/src/lib/globals.d.ts`, or adding it to DefinitelyTyped?